api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java

```diff
@@ -888,37 +889,83 @@ public void updateFamilyName_badRequest() {
 
   @Test
   public void resendWelcomeEmail_messagingException() throws MessagingException {
+    config.access.tiersVisibleToUsers =
```

Comment:
please add a comment explaining why this is necessary

Reply:
please also restore the deprecated versions of these tests

Reply:
This functionality will no longer be used as of the next production release, so we decided that the tests do not provide enough utility for the time needed to bring them back.